std/experimental/ndslice/slice.d

```diff
@@ -383,6 +416,17 @@ pure nothrow @nogc unittest
     static assert(isRandomAccessRange!S);
 }
 
+///
+pure nothrow @nogc unittest
```

Comment:
This unit test exists to show off the `AllowDownsize` flag, right? If so it may be useful to add a comment inside the test saying so.